webpack/assets/javascripts/react_app/components/FactCharts/FactChartActions.js

```diff
@@ -1,16 +1,7 @@
 import {
-  FACT_CHART,
   FACT_CHART_MODAL_OPEN,
   FACT_CHART_MODAL_CLOSE,
 } from './FactChartConstants';
-import { get } from '../../redux/API';
-
-export const getChartData = (url, id) =>
-  get({
```

Comment:
too simple for being in a separate action file - moved to `index.js`

Reply:
I think it should still stay here so it will be clear what are the all fact charts actions are.

Reply:
I spoke about it offline with @sharvit,
imo it makes sense to move it to the index file since it is actually an API action consumed here,
and not a custom fact chart action.

Reply:
after we discussed it together offline, I moved it to the action file since there are actually two actions here. if it was only the get API action I would leave it here.